src/pages/LandingPage.js

```diff
@@ -256,8 +256,7 @@ class LandingPage extends React.Component {
       isEmptyState: true,
       isAddTripState: false,
     })
-    // eslint-disable-next-line no-console
-    console.log('hello there')
+    logDebug('hello there')
```

Comment:
Are we starting another PR to remove unnecessary logging?

Reply:
the logDebug only logs when the NODE_ENV is set to "development"